09-intermediate-rails/intro-to-oauth.md

```diff
@@ -1,61 +1,119 @@
-# Introduction to OAuth
+# Introduction to Authentication and OAuth
 
-Today we will introduce a technology called *OAuth*, which we will use to allow users to log into our Rails app. OAuth is short for **Open Authentication**, and it is an open standard that describes how users can use one service to log into another. If you've ever logged into a website using Facebook, you've used OAuth.
+## Learning Goals
 
+By the end of this lesson, students should be able to answer the following questions or do the following things:
```

Comment:
I would take off, do the following things, and just make the vocab learning goals a question as well, since none of these are things the student is asked to do.